### PR TITLE
clarify the payment_method_id in project.html.markdown

### PR DESCRIPTION
**Pull request by larrymapacket**
the 'payment_method_id' needs to belong to the same 'organization_id'

### CODE REVIEW

File: website/docs/r/project.html.markdown [hunk 1]
```diff
@@ -55,7 +55,7 @@ The `bgp_config` block supports:
 The following attributes are exported:
 
 * `id` - The unique ID of the project
-* `payment_method_id` - The UUID of payment method for this project.
+* `payment_method_id` - The UUID of payment method for this project,needing to belong to the same organization_id.
```
Comment (johnstudarus) [inline website/docs/r/project.html.markdown]: ^, needing^ which needs

Comment (t0mk) [reply website/docs/r/project.html.markdown]: @larrymapacket, I would reword to
"The UUID of payment method for this project, which needs to belong to the same organization (default, or the one passed with organizaition_id)."

**Comment by t0mk** [comment]
@larrymapacket Thank you for the PR!
